core/model/modx/processors/system/event/getlist.class.php

```diff
@@ -11,60 +11,94 @@
  * @package modx
  * @subpackage processors.system.event
  */
-class modSystemEventGetListProcessor extends modProcessor {
-    public function checkPermissions() {
+class modSystemEventGetListProcessor extends modProcessor
+{
+    public function checkPermissions()
+    {
         return $this->modx->hasPermission('events');
     }
-    public function getLanguageTopics() {
+
+    public function getLanguageTopics()
+    {
         return array('events');
     }
-    public function initialize() {
+
+    public function initialize()
+    {
         $this->setDefaultProperties(array(
             'start' => 0,
             'limit' => 10,
-            'sort' => 'groupname ASC, name',
+            'sort' => 'modEvent_groupname ASC, modEvent_name',
             'dir' => 'ASC',
         ));
         return true;
     }
-    
-    public function process() {
+
+    public function process()
+    {
         $data = $this->getData();
-        
+
         $list = array();
         /** @var modEvent $event */
         foreach ($data['results'] as $event) {
             $eventArray = $event->toArray();
 
+            if ($eventArray['plugins']) {
+                $c = $this->modx->newQuery('modPlugin');
+                $c->leftJoin('modPluginEvent', 'modPluginEvent', 'modPluginEvent.pluginid = modPlugin.id');
+                $c->sortby('modPluginEvent.priority', 'ASC');
+                $c->where(['modPluginEvent.event' => $eventArray['name']]);
```

Comment:
While i would love to use the short array syntax, this would break compatibility to PHP 5.3.x